libgalois/include/katana/analytics/Plan.h

```diff
@@ -12,13 +12,27 @@ enum Architecture {
   kDistributed
 };
 
+/// The base class for abstract algorithm execution plans.
+///
+/// Execution plans contain any tuning parameters the abstract algorithm requires. In general, this will include
+/// selecting the concrete algorithm to use and providing it's tuning parameters. Plans do not affect the result (beyond
+/// potentially returning a different valid result in the case of non-deterministic algorithms). For example, the tile
```

Comment:
We don't expect the tile size to result in different results for all the algorithms. I would just say "..case of non-deterministic algorithm or floating-point operations"

Reply:
This paragraph doesn't imply that the tile size will change the results to me. I'm not sure what you are responding to.

Reply:
I read more carefully and I understand. Fixed.